apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -74,7 +78,17 @@
                     console.warn("Prevented a possible double submit event");
                 } else {
                     e.preventDefault();
-
+                    <%
+                        if (reCaptchaEnabled) {
+                    %>
+                    if (!grecaptcha.getResponse()) {
+                        event.preventDefault(); //prevent form submit
```

Comment:
Format comments.

Reply:
Was fixed in https://github.com/wso2/identity-apps/pull/3301